src/fixed.jl

```diff
@@ -16,27 +11,40 @@ function MP.polynomialtype(basis::AbstractPolynomialVectorBasis{PT}, T::Type) wh
 end
 function MP.polynomial(f::Function, basis::AbstractPolynomialVectorBasis)
     return MP.polynomial(mapreduce(
-        ip -> f(ip[1]) * ip[2], MA.add!, enumerate(basis.polynomials)))
+        ip -> f(ip[1]) * ip[2], MA.add!, enumerate(basis.elements)))
 end
 
 function MP.polynomial(Q::AbstractMatrix, basis::AbstractPolynomialVectorBasis,
                        T::Type)
     n = length(basis)
     @assert size(Q) == (n, n)
-    return MP.polynomial(mapreduce(row -> basis.polynomials[row] *
-        mapreduce(col -> Q[row, col] * basis.polynomials[col], MA.add!, 1:n),
+    return MP.polynomial(mapreduce(row -> basis.elements[row] *
+        mapreduce(col -> Q[row, col] * basis.elements[col], MA.add!, 1:n),
         MA.add!, 1:n), T)
 end
 
 """
     struct FixedPolynomialBasis{PT<:MP.AbstractPolynomialLike, PV<:AbstractVector{PT}} <: AbstractPolynomialBasis
-        polynomials::PV
+        elements::PV
     end
 
-Polynomial basis with the polynomials of the vector `polynomials`.
+Polynomial basis with the polynomials of the vector `elements`.
 For instance, `FixedPolynomialBasis([1, x, 2x^2-1, 4x^3-3x])` is the Chebyshev
 polynomial basis for cubic polynomials in the variable `x`.
 """
 struct FixedPolynomialBasis{PT<:MP.AbstractPolynomialLike, PV<:AbstractVector{PT}} <: AbstractPolynomialVectorBasis{PT, PV}
-    polynomials::PV
+    elements::PV
 end
+
+function MP.coefficients(p, Basis::Type{<:AbstractPolynomialBasis})
+    basis = maxdegree_basis(Basis, variables(p), maxdegree(p))
```

Comment:
Add your convincing argument of https://github.com/JuliaAlgebra/MultivariateBases.jl/issues/4#issuecomment-610032166 as comment.